code/datums/diseases/transformation.dm

```diff
@@ -83,19 +83,16 @@
 /datum/disease/transformation/proc/replace_banned_player(var/mob/living/new_mob) // This can run well after the mob has been transferred, so need a handle on the new mob to kill it if needed.
 	set waitfor = FALSE
 
+	affected_mob.ghostize(TRUE,TRUE)
```

Comment:
Doesn't ghostize(TRUE,... allow for people to reenter the body?

Reply:
It shouldn't; ghost enter checks if the mob has a key/client before assigning it, so it should be ok.